rpm/sailfish-office.spec

```diff
@@ -44,6 +44,7 @@ Group: System/Base
 %{_datadir}/translations/*.qm
 %{_datadir}/dbus-1/interfaces/
 %{_datadir}/dbus-1/services/
+%{_datadir}/lipstick/notificationcategories/x-sailfish.sailfish-office.error.conf
 
```

Comment:
For in app notifications this is not needed, we don't want the notification to show up in events view.